docs/modules/ROOT/nav-end.adoc

```diff
@@ -14,7 +14,8 @@
 *** xref:architecture/cr/camel-catalog.adoc[CamelCatalog]
 ** xref:architecture/runtime.adoc[Runtime]
 ** xref:architecture/traits.adoc[Traits]
-* xref:apis/camel.adoc[API]
+* xref:apis/camel-k.adoc[Camel-K API]
```

Comment:
Maybe we better have:
```
* API
** xref:apis/camel-k.adoc[Camel-K]
** xref:apis/kamelets.adoc[Kamelets]
```

Reply:
If we keep the Kamelets API page under camel-k, yes.... I don't think anyone has commented on my proposal to move the kamelets documentation, including the kamelets API, to the kamelets subproject.

I might have fixed the preview problem, there was a old blog link.

Reply:
IMO we should still keep the Kamelets documentation under Camel K, at least until we are in beta API version. About the new link, I am not able to see anyone but the first one generated by this PR.

Reply:
Added the API heading.